src/sentry/relay/config/metric_extraction.py

```diff
@@ -165,28 +164,6 @@ def _get_alert_metric_specs(
                     )
                     specs.append(spec)
 
-            # In case the query has an environment, we want to extract with the old environment logic, since we found
```

Comment:
We do not need this block anymore since `_convert_snuba_query_to_metrics` will return all versions of metric specs.